projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -163,15 +239,6 @@ class_exists( 'Jetpack_AMP_Support' )
 
 		if (
 			! empty( $output ) &&
-			/**
-			 * Allow third-party plugins or themes to force-enable Carousel.
-			 *
-			 * @module carousel
-			 *
-			 * @since 1.9.0
-			 *
-			 * @param bool false Should we force enable Carousel? Default to false.
-			 */
```

Comment:
Ah shoot, I missed adding this one back. What I’d done was move these to be function headers throughout the file without realising the implications of what I was doing. @jeherve [flagged this earlier too](https://github.com/Automattic/jetpack/pull/22770#discussion_r805727398) - that the filters needs a docblock so that the codex can be automatically generated for [developer.jetpack.com](http://developer.jetpack.com/). I can fix this one in another commit. Thanks!